### PR TITLE
[IMP] accounting: Bank Synchronization: automatic import

### PR DESCRIPTION
**Pull request by chiaraprattico**
taskid: 3046370

### CODE REVIEW

File: content/applications/finance/accounting/bank/feeds/bank_synchronization.rst [hunk 1]
```diff
@@ -38,49 +33,49 @@ following address:
...
 .. note::
    If you have any issues during your first synchronization, please verify that your
-   web browser doesn't block pop-ups and that your adblocker is disabled.
+   web browser doesn't block pop-ups and that your ad-blocker is disabled.
+
+.. important::
+   When choosing the date for the first bank statements synchronization, pick the actual date where
+   you start working on your Odoo accounting database. For example, if you import your closing
+   balance in Odoo on the 31.12.2022 and you start recording accounting transactions on the
+   01.01.2023, your synchronization date should be 01.01.2023.
```
Comment (LoredanaLrpz) [inline content/applications/finance/accounting/bank/feeds/bank_synchronization.rst:52]: I'd rephrase this a bit:
```suggestion
.. important::
   When choosing the date for the first bank statement synchronization, pick the date of when you
   start working on your Odoo accounting database. For example, if you import your closing balance 
   in Odoo on 31.12.2022, but start recording accounting transactions on 01.01.2023, your
   synchronization date should be 01.01.2023.

```
The most important parts in this suggestion are:
- no "the" before the dates
- ",but start" -> I think it makes more sense that way

File: content/applications/finance/accounting/bank/feeds/bank_synchronization.rst [hunk 2]
```diff
@@ -38,49 +33,49 @@ following address:
...
+
```
Comment (LoredanaLrpz) [inline content/applications/finance/accounting/bank/feeds/bank_synchronization.rst:53]: This line should go :)

File: content/applications/finance/accounting/bank/feeds/bank_synchronization.rst [hunk 3]
```diff
@@ -38,49 +33,49 @@ following address:
...
 
 During your first synchronization, you will be asked for a phone number to secure your account.
 The reason we ask for such information is that we don't want your data falling into the wrong
 hands. Therefore, if we detect suspicious activity on your account, we block all requests coming
 from your account, and you need to reactivate it using that phone number.
```
Comment (LoredanaLrpz) [inline content/applications/finance/accounting/bank/feeds/bank_synchronization.rst:58]:
```suggestion
You must provide a phone number during your first synchronization to secure your account. We ask for
such information because we don't want your data falling into the wrong hands. Therefore, if we
detect suspicious activities on your account, we block all requests coming from your account, and 
you need to reactivate it using that phone number.
```

File: content/applications/finance/accounting/bank/feeds/bank_synchronization.rst [hunk 4]
```diff
@@ -38,49 +33,49 @@ following address:
...
-The third-party provider may request more information in order to connect with your bank institution.
+The third-party provider may request more information in order to connect with your
+bank institution.
 This information is not stored on Odoo's servers.
```
Comment (LoredanaLrpz) [inline content/applications/finance/accounting/bank/feeds/bank_synchronization.rst:61]: I'd put this as a note

```suggestion

.. note::
   This information is not stored on Odoo's servers.
```

File: content/applications/finance/accounting/bank/feeds/bank_synchronization.rst [hunk 5]
```diff
@@ -38,49 +33,49 @@ following address:
...
+   01.01.2023, your synchronization date should be 01.01.2023.
+
 
 During your first synchronization, you will be asked for a phone number to secure your account.
 The reason we ask for such information is that we don't want your data falling into the wrong
 hands. Therefore, if we detect suspicious activity on your account, we block all requests coming
 from your account, and you need to reactivate it using that phone number.
-The third-party provider may request more information in order to connect with your bank institution.
+The third-party provider may request more information in order to connect with your
+bank institution.
```
Comment (LoredanaLrpz) [inline content/applications/finance/accounting/bank/feeds/bank_synchronization.rst:60]: I just added one line above to create another § :)
```suggestion

The third-party provider may request more information in order to connect with your
bank institution.
```

File: content/applications/finance/accounting/bank/feeds/bank_synchronization.rst [hunk 6]
```diff
@@ -95,14 +90,14 @@ Synchronization in error
...
-:menuselection:`Accounting --> Configuration --> Online Synchronization`, select the connection
+:menuselection:`Configuration --> Online Synchronization`, select the connection
 that failed, and copy the error description and the reference.
 
 Synchronization disconnected
 ----------------------------
 
 If your connection with the proxy is disconnected, you can reconnect with the proxy using the
-*Reconnect* button.
+:guilabel: `fetch account` button.
```
Comment (LoredanaLrpz) [inline content/applications/finance/accounting/bank/feeds/bank_synchronization.rst:100]: There's a space here that breaks guilabel
```suggestion
:guilabel:`fetch account` button.
```

File: content/applications/finance/accounting/bank/feeds/bank_synchronization.rst [hunk 7]
```diff
@@ -121,8 +116,8 @@ Users who have created a database before December 2020 need to install the new m
 use the new functionalities.
 
 To do so, go to :menuselection:`Apps --> Update Apps List`, remove the default filter in the search
-bar and type: **account_online_synchronization**. You can then click on the *Install* button to
-install the new module.
+bar and type: **account_online_synchronization**. You can then click on the :guilable:`Install`
```
Comment (LoredanaLrpz) [inline content/applications/finance/accounting/bank/feeds/bank_synchronization.rst:119]: Usually, text to type is between ` and typo in guilabel
```suggestion
bar and type `account_online_synchronization`. You can then click on the :guilabel:`Install`
```

File: content/applications/finance/accounting/bank/feeds/bank_synchronization.rst [hunk 8]
```diff
@@ -121,8 +116,8 @@ Users who have created a database before December 2020 need to install the new m
...
+button to install the new module.
 
 .. image:: bank_synchronization/online-sync-module.png
```
Comment (LoredanaLrpz) [inline content/applications/finance/accounting/bank/feeds/bank_synchronization.rst:122]: the image is broken, if you've deleted it, then the lines should be removed (also, there's no image with that name in the bank_sync folder, only `online-sync-sync-now-dashboard.png`
So I'd suggest:
```suggestion
.. image:: bank_synchronization/online-sync-sync-now-dashboard.png
```

File: content/applications/finance/accounting/bank/feeds/bank_synchronization.rst [hunk 9]
```diff
@@ -133,12 +128,12 @@ Finally, make sure all your users refresh their Odoo page by pressing CTRL+F5.
 .. Note::
 
    - All previous synchronizations are disconnected during the installation and won't work anymore.
-     You can find them directly in the synchronization menu (:menuselection:`Accounting -->
-     Configuration --> Online Synchronization`). It is not possible to resynchronize these
-     connections; you have to make new ones.
-   - Please do not uninstall *account_online_sync* which is the previous module for online
+     You can find them directly in the synchronization menu (:menuselection:` Configuration -->
+     Online Synchronization`). It is not possible to resynchronize these connections; you have to
+     make new ones.
```
Comment (LoredanaLrpz) [inline content/applications/finance/accounting/bank/feeds/bank_synchronization.rst:133]: The :menuselection: is broken
```suggestion
     You can find them directly in the synchronization menu (:menuselection:`Configuration -->
     Online Synchronization`). It is not possible to resynchronize these connections; you have to
     make new ones.
```

File: content/applications/finance/accounting/bank/feeds/bank_synchronization.rst (hunk 8 above)
Comment (LoredanaLrpz) [reply content/applications/finance/accounting/bank/feeds/bank_synchronization.rst:122]: after building the doc page, that image might actually not suit the § x)

File: content/applications/finance/accounting/bank/feeds/bank_synchronization.rst [hunk 10]
```diff
@@ -148,14 +143,15 @@ The synchronization is not working in real-time. Is that normal?
...
+and fetch transactions through :menuselection:`Configuration --> Online Synchronization`.
+Some providers only allow one refresh per day, so it is possible that clicking on
+:guilabel:`Synchronize Now` does not get your latest transactions if you already performed such
+action earlier in the day.
 
 A transaction can be visible on your bank account, but not be fetched if it has the status
-*Pending*. Only transactions with the *Posted* status will be retrieved. If it is not *Posted* yet,
-you will have to wait until the status changes.
+**Pending**. Only transactions with the **Posted** status will be retrieved. If it is not **Posted**
+yet, you will have to wait until the status changes.
```
Comment (LoredanaLrpz) [inline content/applications/finance/accounting/bank/feeds/bank_synchronization.rst:154]: Wouldn't those be guilabel? (I think the status appear on the app. Not sure about the last posted at all so I left it as ** **
```suggestion
A transaction can be visible on your bank account but not be fetched if it has the status
:guilabel:`Pending`. Only transactions with the :guilabel:`Posted` status will be retrieved. If the
transaction is not **Posted** yet, you will have to wait until the status changes.
```

File: content/applications/finance/accounting/bank/feeds/bank_synchronization.rst [hunk 11]
```diff
@@ -188,8 +184,9 @@ Why don't I see any transactions?
 ---------------------------------
 
 During your first synchronization, you selected the bank accounts you decided to synchronize with
-Odoo. If you didn't synchronize any of your accounts, you can go to :menuselection:`Accounting -->
-Configuration --> Online Synchronization` to click on the *Fetch Accounts* button on the connection.
+Odoo. If you didn't synchronize any of your accounts, you can go to
+:menuselection:`Configuration --> Online Synchronization` to click on the
+:guilabel:`Fetch Accounts` button on the connection.
```
Comment (LoredanaLrpz) [inline content/applications/finance/accounting/bank/feeds/bank_synchronization.rst:189]:
```suggestion
Odoo. If you didn't synchronize any of your accounts, you can go to :menuselection:`Configuration 
--> Online Synchronization` to click on the :guilabel:`Fetch Accounts` button on the connection.
```

**Review by LoredanaLrpz — CHANGES_REQUESTED** [review]
There you go =D

**Review by LoredanaLrpz — APPROVED** [review]

File: content/applications/finance/accounting/bank/feeds/bank_synchronization.rst [hunk 12]
```diff
@@ -38,49 +33,50 @@ following address:
...
-During your first synchronization, you will be asked for a phone number to secure your account.
-The reason we ask for such information is that we don't want your data falling into the wrong
-hands. Therefore, if we detect suspicious activity on your account, we block all requests coming
-from your account, and you need to reactivate it using that phone number.
-The third-party provider may request more information in order to connect with your bank institution.
+.. important::
+   When choosing the date for the first bank statement synchronization, pick the date of when you
+   start working on your Odoo accounting database. For example, if you import your closing
+   balance in Odoo on the 31.12.2022 and you start recording accounting transactions on the
+   01.01.2023, your synchronization date should be 01.01.2023.
```
Comment (xpl-odoo) [inline content/applications/finance/accounting/bank/feeds/bank_synchronization.rst:53]:
```suggestion
   When choosing the date for the first bank statement synchronization, pick the date when you
   start working on your Odoo accounting database. For example, if you import your closing
   balance in Odoo on 31/12/2022 and you start recording accounting transactions on the
   01/01/2023, your synchronization date should be 01/01/2023.
```

- I would use the date format of Odoo with `/` instead of `.` as separators
- I would clarify: _pick the date when you start working on your Odoo accounting database_. You could say, _'pick the date when you start recording accounting transactions on your Odoo database'_.

File: content/applications/finance/accounting/bank/feeds/bank_synchronization.rst [hunk 13]
```diff
@@ -38,49 +33,50 @@ following address:
...
+bank institution.
 This information is not stored on Odoo's servers.
 
 By default, transactions fetched from an online source are grouped inside the same statement, and
 one bank statement is created per month. You can change the bank statement creation periodicity
 in your journal settings.
 
-You can find all your synchronizations by going to :menuselection:`Accounting --> Configuration -->
+You can find all your synchronizations by going to :menuselection:`Configuration -->
 Online Synchronization`.
```
Comment (xpl-odoo) [inline content/applications/finance/accounting/bank/feeds/bank_synchronization.rst:69]:
```suggestion
You can find all your synchronizations by going to :menuselection:`Accounting Dashboard --> Configuration -->
Accounting: Online Synchronization`.
```
We usually keep the full path as people might not check the previous sections and jump straight to that info.
Also, I added the sub-menu, but feel free to remove it if you don't like it ;)

File: content/applications/finance/accounting/bank/feeds/bank_synchronization.rst [hunk 14]
```diff
@@ -38,49 +33,50 @@ following address:
...
-
-Or you can go to :menuselection:`Accounting --> Configuration --> Online Synchronization`,
-select your institution and then click on the *Fetch Transactions* button.
+If you wish, you can synchronize manually by clicking on the :guilabel:`Synchronize Now` button on
+the dashboard.
 
-.. image:: bank_synchronization/online-sync-form-view.png
-   :align: center
-   :alt: Online Synchronization Form view
+Or you can go to :menuselection:`Configuration --> Online Synchronization`,
```
Comment (xpl-odoo) [inline content/applications/finance/accounting/bank/feeds/bank_synchronization.rst:78]:
```suggestion
Or you can go to :menuselection:`Accounting Dashboard --> Configuration --> Accounting: Online Synchronization`,
```

File: content/applications/finance/accounting/bank/feeds/bank_synchronization.rst [hunk 15]
```diff
@@ -95,14 +91,14 @@ Synchronization in error
 ------------------------
 
 To report a connection error to the `Odoo support <https://www.odoo.com/help>`_, go to
-:menuselection:`Accounting --> Configuration --> Online Synchronization`, select the connection
+:menuselection:`Configuration --> Online Synchronization`, select the connection
```
Comment (xpl-odoo) [inline content/applications/finance/accounting/bank/feeds/bank_synchronization.rst:94]:
```suggestion
:menuselection:`Accounting Dashboard --> Configuration --> Accounting: Online Synchronization`, select the connection
```

File: content/applications/finance/accounting/bank/feeds/bank_synchronization.rst [hunk 16]
```diff
@@ -121,24 +117,18 @@ Users who have created a database before December 2020 need to install the new m
...
 
 .. Note::
 
    - All previous synchronizations are disconnected during the installation and won't work anymore.
-     You can find them directly in the synchronization menu (:menuselection:`Accounting -->
-     Configuration --> Online Synchronization`). It is not possible to resynchronize these
-     connections; you have to make new ones.
-   - Please do not uninstall *account_online_sync* which is the previous module for online
+   - You can find them directly in the synchronization menu
+     (:menuselection:`Configuration --> Online Synchronization`). It is not possible to
```
Comment (xpl-odoo) [inline content/applications/finance/accounting/bank/feeds/bank_synchronization.rst:127]:
```suggestion
     (:menuselection:`Accounting Dashboard --> Configuration --> Accounting: Online Synchronization`). It is not possible to
```

File: content/applications/finance/accounting/bank/feeds/bank_synchronization.rst [hunk 17]
```diff
@@ -121,24 +117,18 @@ Users who have created a database before December 2020 need to install the new m
...
 
 To do so, go to :menuselection:`Apps --> Update Apps List`, remove the default filter in the search
-bar and type: **account_online_synchronization**. You can then click on the *Install* button to
-install the new module.
-
-.. image:: bank_synchronization/online-sync-module.png
-   :align: center
-   :alt: Installation button of the account_online_synchronization module
-
+bar and type `account_online_synchronization`. You can then click on the :guilabel:`Install`.
```
Comment (xpl-odoo) [inline content/applications/finance/accounting/bank/feeds/bank_synchronization.rst:120]:
```suggestion
bar and type `account_online_synchronization`. You can then click on :guilabel:`Install`.
```

File: content/applications/finance/accounting/bank/feeds/bank_synchronization.rst [hunk 18]
```diff
@@ -121,24 +117,18 @@ Users who have created a database before December 2020 need to install the new m
...
 
    - All previous synchronizations are disconnected during the installation and won't work anymore.
-     You can find them directly in the synchronization menu (:menuselection:`Accounting -->
-     Configuration --> Online Synchronization`). It is not possible to resynchronize these
-     connections; you have to make new ones.
-   - Please do not uninstall *account_online_sync* which is the previous module for online
+   - You can find them directly in the synchronization menu
+     (:menuselection:`Configuration --> Online Synchronization`). It is not possible to
+     resynchronize these connections; you have to make new ones.
+   - Please do not uninstall **account_online_sync** which is the previous module for online
```
Comment (xpl-odoo) [inline content/applications/finance/accounting/bank/feeds/bank_synchronization.rst:129]:
```suggestion
   - Please do not uninstall **account_online_sync**, which is the previous module for online
```

File: content/applications/finance/accounting/bank/feeds/bank_synchronization.rst [hunk 19]
```diff
@@ -121,24 +117,18 @@ Users who have created a database before December 2020 need to install the new m
...
      synchronization. The new one overrides it.
-   - By default *account_online_synchronization* is installed automatically with Accounting.
+   - By default **account_online_synchronization** is installed automatically with Accounting.
```
Comment (xpl-odoo) [inline content/applications/finance/accounting/bank/feeds/bank_synchronization.rst:131]:
```suggestion
   - By default, `account_online_synchronization` is installed automatically with Accounting.
```

File: content/applications/finance/accounting/bank/feeds/bank_synchronization.rst (hunk 18 above)
Comment (xpl-odoo) [reply content/applications/finance/accounting/bank/feeds/bank_synchronization.rst:129]:
```suggestion
   - Do not uninstall `account_online_sync`, which is the previous module for online
```
Just a personal preference, but bold is fine as well ;)

File: content/applications/finance/accounting/bank/feeds/bank_synchronization.rst [hunk 20]
```diff
@@ -148,14 +138,15 @@ The synchronization is not working in real-time. Is that normal?
 
 The process is not intended to work in real-time as third party providers synchronize your accounts
 at different intervals. To force the synchronization and fetch the statements, go to your
-*Accounting dashboard*, and click on the *Synchronize Now* button. You can also synchronize and
-fetch transactions through :menuselection:`Accounting --> Configuration --> Online Synchronization`.
-Some providers only allow one refresh per day, so it is possible that clicking on *Synchronize Now*
-does not get your latest transactions if you already performed such action earlier in the day.
+Accounting dashboard, and click on the :guilabel:`Synchronize Now` button. You can also synchronize
```
Comment (xpl-odoo) [inline content/applications/finance/accounting/bank/feeds/bank_synchronization.rst:141]:
```suggestion
:guilabel:`Accounting Dashboard`, and click on the :guilabel:`Synchronize Now` button. You can also synchronize
```

File: content/applications/finance/accounting/bank/feeds/bank_synchronization.rst [hunk 21]
```diff
@@ -148,14 +138,15 @@ The synchronization is not working in real-time. Is that normal?
...
+and fetch transactions through :menuselection:`Configuration --> Online Synchronization`.
```
Comment (xpl-odoo) [inline content/applications/finance/accounting/bank/feeds/bank_synchronization.rst:142]:
```suggestion
and fetch transactions through :menuselection:`Accounting Dashboard --> Configuration --> Accounting: Online Synchronization`.
```

File: content/applications/finance/accounting/bank/feeds/bank_synchronization.rst [hunk 22]
```diff
@@ -188,8 +179,9 @@ Why don't I see any transactions?
 ---------------------------------
 
 During your first synchronization, you selected the bank accounts you decided to synchronize with
-Odoo. If you didn't synchronize any of your accounts, you can go to :menuselection:`Accounting -->
-Configuration --> Online Synchronization` to click on the *Fetch Accounts* button on the connection.
+Odoo. If you didn't synchronize any of your accounts, you can go to
+:menuselection:`Configuration --> Online Synchronization` to click on the :guilabel:`Fetch Accounts`
```
Comment (xpl-odoo) [inline content/applications/finance/accounting/bank/feeds/bank_synchronization.rst:183]:
```suggestion
:menuselection:`Accounting Dashboard --> Configuration --> Online Synchronization` to click on the :guilabel:`Fetch Accounts`
```

File: content/applications/finance/accounting/bank/feeds/bank_synchronization.rst [hunk 23]
```diff
@@ -95,14 +91,14 @@ Synchronization in error
...
-:menuselection:`Accounting --> Configuration --> Online Synchronization`, select the connection
+:menuselection:`Configuration --> Online Synchronization`, select the connection
 that failed, and copy the error description and the reference.
 
 Synchronization disconnected
 ----------------------------
 
 If your connection with the proxy is disconnected, you can reconnect with the proxy using the
-*Reconnect* button.
+:guilabel:`fetch account` button.
```
Comment (xpl-odoo) [inline content/applications/finance/accounting/bank/feeds/bank_synchronization.rst:101]: Try to be consistent: 'fetch account' or 'Fetch Account' button, not both :) I also have seen 'Fetch Account**s**'

File: content/applications/finance/accounting/bank/feeds/bank_synchronization.rst [hunk 24]
```diff
@@ -199,9 +191,9 @@ database, please `submit a support ticket <https://www.odoo.com/help>`_.
 How can I update my bank credentials?
 -------------------------------------
 
-You can update your credentials by going to :menuselection:`Accounting --> Configuration --> Online
-Synchronization`, open the connection you want to update your credentials and click on the *Update
-Credentials* button.
+You can update your credentials by going to :menuselection:`Configuration --> Online
+Synchronization`,open the connection you want to update your credentials and click on the
+:guilabel:`Update Credentials` button.
```
Comment (xpl-odoo) [inline content/applications/finance/accounting/bank/feeds/bank_synchronization.rst:196]:
```suggestion
You can update your credentials by going to :menuselection:`Accounting Dashboard: Configuration --> Accounting: Online
Synchronization`, open the connection you want to update your credentials and click on the
:guilabel:`Update Credentials` button.
```

**Review by xpl-odoo — CHANGES_REQUESTED** [review]
Nice job!

Mostly nitpicks ;)

**Review by AntoineVDV — APPROVED** [review]
@robodoo r+
